src/ensembl/src/shared/components/panel/Panel.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import classNamesMerger from 'classnames';
+
+import closeIcon from 'static/img/shared/close.svg';
+
+import styles from './Panel.scss';
+
+export type PanelProps = {
+  header: string | JSX.Element;
+  children: JSX.Element;
+  classNames?: {
+    panel?: string;
+    header?: string;
+    body?: string;
+    closeButton?: string;
+  };
+  onClose?: () => void;
+};
+
+const Panel = (props: PanelProps) => {
+  const { header, onClose, classNames } = props;
+
+  const panelClassNames = classNames
+    ? classNamesMerger(classNames.panel, styles.panel)
```

Comment:
Here, and in other mergers: shouldn't the order of the arguments be the opposite? Shouldn't the class passed as a prop be able to override the default class of the Panel?

Reply:
That's right. I had a different issue so I tried the other way around but forgot to revert it back.